packages/fxa-auth-server/config/index.js

```diff
@@ -742,6 +742,26 @@ const conf = convict({
       env: 'FLOW_ID_EXPIRY',
     },
   },
+  statsd: {
+    host: {
+      default: 'localhost',
```

Comment:
It doesn't look like there's a statsd server running at port 8125 in `servers.json` for pm2. Should we add one?

Reply:
`nc -lvu 127.0.0.1 8125` was the extent of me verifying the output. :D  I don't know if a server would add much value since we don't do anything with those stats locally.  Unless you want something more than just printing out the stats or even an actual telegraf server to ingest the data?

Reply:
Yeah, probably not worth over-doing. I think failing gracefully and handling any potential errors are the more valuable things here.